repository/rest-proxy/operator/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+    app: rest-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rest-proxy
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: rest-proxy
+    spec:
+      containers:
+      - name: rest-proxy
+        image: mesosphere/cp-kafka-rest:5.3.0
```

Comment:
as we talked, lets test with confluent image and not mesosphere image

Reply:
Done. Changed the image to Confluentic one, and now using the latest version v5.3.2.